packages/mobile/android/app/build.gradle

```diff
@@ -56,7 +56,7 @@ project.ext.envConfigFiles = [
     releasecandidaterelease: ".env.prod"
 ]
 
-apply from: "../../node_modules/@sentry/react-native/sentry.gradle"
+apply from: "../../../../node_modules/@sentry/react-native/sentry.gradle"
```

Comment:
This is weird, why is this required?

Reply:
when the reinstall happened, looks like react-native/sentry got moved to the root node_modules

Reply:
Wouldn't it only have to go up 3 levels? maybe I'm just looking at it wrong

Reply:
so i could be wrong, but its up 2 to go from build.gradle to mobile/node_modules, and then two more to go to root modules, cause its ../packages ../../root